data/language/en-GB.txt

```diff
@@ -3456,6 +3456,7 @@ STR_3443    :Page 4
 STR_3444    :Page 5
 STR_3445    :Set Patrol Area
 STR_3446    :Cancel Patrol Area
+STR_3447    :{BLACK}No mechanics are hired...
```

Comment:
I'd recommend removing the ellipsis and replacing it with either an exclamation point or simple full stop. Ellipsis are generally used in RCT when something can't be done and the reason is unknown or unspecified (e.g. "Unable to place this here...").

In this message, all the information is already there, so the ellipsis isn't needed.